.github/actions/validate-apiml-healthy/action.yml

```diff
@@ -1,5 +1,5 @@
 name: "Validate APIML Healthy"
-description: "Validates APIML health using Gateway's health status"
```

Comment:
I guess this name needs to change, since it's using more than just the gateway health status.
But I suppose this is going to be replaced by cloud gateway's health status rather than zaas.